contributors/design-proposals/standardization-of-development-process.md

```diff
@@ -0,0 +1,244 @@
+categorization tricky. While not all community members might not be comfortable
+using Git directly, I believe it is imperative that as a community we work to
+educate people on a standard set of tools so they can take their experience to
+other projects they may decide to work on in the future.
+
+It is my hope that by increasing the requirements for proposing a change to
+Kubernetes we can prevent important work like generating release notes from
+falling out of the release cycle and in general make working with and speaking
+about Kubernetes an easier and more pleasant experience.
+
+[road to Go 2]: https://blog.golang.org/toward-go2
+
+## Detailed Design
+
+### RFC Template
+
+### Git and GitHub Implementation
+
+Practically an RFC would be implemented as a pull request to a central repository
+with the following example structure
+
+```
+├── 0000-rfc-template.md
+├── CODEOWNERS
+├── index.md
+├── sig-architecture
+├── sig-network
+│   └── kube-dns
+├── sig-node
+│   └── kubelet
+├── sig-release
+├── sig-storage
+├── unsorted-to-be-used-by-newcomers-only
+└── wg-resource-management
+```
+
+where each SIG or working group is given a top level directory with components
+maintained by the SIG listed in sub directories. For newcomers to the community
+an `unsorted-to-be-used-by-newcomers-only` directory may be used before an RFC
+can be properly routed to a SIG although hopefully if discussion for a potential
+RFC begins on the mailing lists proper routing information will be provided to
+the RFC author. Additionally a top level index of RFCs may be helpful for people
+looking for a complete list of RFCs. There should be basic CI to ensure that a
+`index.md` remains up to date.
+
+Ideally no work would begin on an RFC before it has been approved by the
+responsible SIG or working group. In order to help combat the explosion of GitHub
+issues against `kubernetes/kubernetes` it is further proposed for each chartered
+SIG to create a top level repository within the `kubernetes` GitHub organization
+where implementation issues would be filed and linked to in the RFC.
+
+### RFC Editor Role
+
+Taking a cue from the [Python PEP process][], I believe that a group of RFC editors
+will be required to make this process successful; the job of an RFC editor is
+likely very similar to the [PEP editor responsibilities][] and will hopefully
+provide another opportunity for people who do not write code daily to contribute
+to Kubernetes.
+
+[Python PEP process]: https://www.python.org/dev/peps/pep-0001/
+[PEP editor responsibilities]: https://www.python.org/dev/peps/pep-0001/#pep-editor-responsibilities-workflow
+
+### Important Metrics
+
+I believe the primary metrics which would signal the success or failure of the RFC
+process would be
+
+- distribution of RFC merge times
+- RFC rejection rate
+- distribution of time between RFC acceptance and implementation issue creation
+- PRs referencing an RFC merged per week
+
+### Customization of the RFC Process
```

Comment:
My gut is that the most value here is when we have things that cross SIGs.  As such, I'm not sure how "opting out" will work.  I would think that proposals that don't cross SIG boundaries would be the exception and not the rule.